src/storage/s3/storage.c

```diff
@@ -586,6 +587,21 @@ typedef struct StorageS3InfoListData
     void *callbackData;                                             // User-supplied callback data
 } StorageS3InfoListData;
 
+// Helper to convert S3 not-quite-ISO-8601 times to time_t
```

Comment:
I think we should define what "not-quite-ISO-8601" actually means - it seems to me it only accepts YYYY-MM-DDTHH:MM:SSZ format (optional .msec after SS).

Reply:
Updated comment.